File: comms/email/fetch-email-operation/fetch-email-operation.cs
using TwilioComms;
using System.Threading.Tasks;

public partial class Examples
{
    public async Task Example() {
        var client = new TwilioCommsClient(
            accountId: "<username>",
            authToken: "<password>"
        );

        await client.Emails.FetchOperationAsync(
            "comms_operation_01h9krwprkeee8fzqspvwy6nq8"
        );
    }

}

File: comms/email/fetch-email-operation/fetch-email-operation.rb
require "twiliocomms"

client = TwilioComms::Client.new(
  account_id: "TWILIO_ACCOUNT_SID",
  auth_token: "TWILIO_AUTH_TOKEN"
)

client.emails.fetch_operation(operation_id: "comms_operation_01h9krwprkeee8fzqspvwy6nq8")
